libtransmission/rpcimpl.cc

```diff
@@ -1149,6 +1149,17 @@ void onPortTested(tr_web::FetchResponse const& web_response)
     auto const& [status, body, primary_ip, did_connect, did_timeout, user_data] = web_response;
     auto* data = static_cast<tr_rpc_idle_data*>(user_data);
 
+    if (auto const addr = tr_address::from_string(primary_ip);
+        data->args_out.find_if<std::string_view>(TR_KEY_ipProtocol) == nullptr)
+    {
+        if (!addr || !addr->is_valid())
+        {
+            tr_idle_function_done(data, fmt::format(_("Couldn't determine IP protocol: {ip}"), fmt::arg("ip", primary_ip)));
```

Comment:
Misleading — empty or unparsable response means we couldn't get the IP address, not the IP protocol. In fact, if client passes the protocol we can't recognize it'll already get back "invalid ip protocol string" error.

Reply:
f061e18 I decided to remove this error message entirely, since an empty IP address is always going to mean something went wrong with the port test, so `status != 200` should catch that and return a meaningful error message.